advisories/github-reviewed/2022/04/GHSA-4pm3-f52j-8ggh/GHSA-4pm3-f52j-8ggh.json

```diff
@@ -63,6 +63,10 @@
       "type": "ADVISORY",
       "url": "https://nvd.nist.gov/vuln/detail/CVE-2022-24847"
     },
+    {
+      "type": "WEB",
+      "url": "https://github.com/geoserver/geoserver/commit/b94a69943992df999d627b21a4ed056fad4569f8"
```

Comment:
This is the meaningful change - provides link to commit that changed JNDI handling for `DataSourceFactoryBean`